### DISCUSSION TITLE
Webcam WebRTC?

### DISCUSSION THREAD

**Discussion by spitfire** [Guilouz/Creality-Helper-Script-Wiki#446] [General]
On my Creality Ender 3 v3 KE I was seeing a process with webrtc in command name. Is there support for viewing camera via WebRTC on K1 (I'm planning to switch to it)? Is there performance between the nebula pad in KE and the pad (does it have its own name) in K1?

**Comment by tent228** [comment]
yes, I was used a webrtc camera by go2rtc.

**Reply by spitfire** [reply]
Just got the Creality K1 camera and it doesn’t support it. Apparently the new model doesn’t offer image using required codec anymore. There’s another discussion in here about it

**Reply by tent228** [reply]
I forwarded the RTSP video stream using the Docker of Go2RTC, and then selected WebRTC (go2RTC) in Mainmail, and that's it.
In addition, I learned on YouTube today that I can directly use a third-party USB camera to successfully drive Logitech's C270. However, it seems a bit slow, and the performance of KE may still be a bit poor.
https://www.youtube.com/watch?v=L92A5V6V4Lo

**Reply by spitfire** [reply]
Like I said I've switched to a K1, and it's (updated, but downgraded version of) camera doesn't support anything besides MJPEG, so I guess I can't do that.
